hack/non-olm-install/install-gitops-operator.sh

```diff
@@ -177,6 +177,7 @@ resources:
   - https://github.com/redhat-developer/gitops-operator/config/crd?ref=$GIT_REVISION&timeout=90s
   - https://github.com/redhat-developer/gitops-operator/config/rbac?ref=$GIT_REVISION&timeout=90s
   - https://github.com/redhat-developer/gitops-operator/config/manager?ref=$GIT_REVISION&timeout=90s
+  - https://raw.githubusercontent.com/redhat-developer/gitops-operator/master/config/rbac/auth_proxy_service.yaml
```

Comment:
@anandf I would suggest instead of adding this file, we should uncomment the auth_proxy_service in the kustomization file and pick it up from there

Please also add `config/prometheus` to this list so that the artifacts necessary for operator metrics are included in non-olm installation

Reply:
Made the suggested changes. Please review.